wdn/templates_4.1/scripts/navigation.js

```diff
@@ -588,10 +591,62 @@ define(['jquery', 'wdn', 'modernizr', 'require'], function($, WDN, Modernizr, re
 		$header.trigger('fixedoffset', [barOffset]);
 	};
 
+	var fixNavButton = function() {
+
+		var $navToggleLabel = $('.wdn-content-slide label[for="wdn_menu_toggle"]');
+
+		//Remove the nav input
+		//$(menuTogSel).hide();
+
+		//handle the label (this should be a button when sends focus)
+		var $navToggleButton = $('<button>');
+		$navToggleButton.html('<span class="wdn-icon-menu" aria-hidden="true"></span><span class="wdn-text-hidden">Menu</span>'); //Make sure they have the same HTML contents
+		$navToggleButton.addClass('wdn-nav-toggle');
+
+		//Handle click events
+		$([$navToggleButton]).each(function(index, $button) {
```

Comment:
This seems weird. Why are you calling each? You already have a jQuery object, so you can go straight to `on`.